src/RepoToolset/Tools.props

```diff
@@ -1,9 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
 <!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
 <Project>
-  <ItemGroup>
-    <PackageReference Include="Microsoft.Net.Compilers" Version="$(MicrosoftNetCompilersVersion)" />
-  </ItemGroup>
+
   <PropertyGroup>
-    <ToolsetCompilerPackageDir>$(NuGetPackageRoot)microsoft.net.compilers\$(MicrosoftNetCompilersVersion)\</ToolsetCompilerPackageDir>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
   </PropertyGroup>
+
+  <Choose>
+    <When Condition="'$(UsingToolMicrosoftNetCompilers)' != 'netcore'">
+      <ItemGroup>
+        <PackageReference Include="Microsoft.Net.Compilers.netcore" Version="$(MicrosoftNetCompilersNetCoreVersion)" />
```

Comment:
Why do I get this package ref if the property is *not* netcore?

Reply:
ah, typo